bash_scripts/cron.jobs

```diff
@@ -1,8 +1,8 @@
 # List of processes run by cron
 # To implement just run "crontab cron.jobs" this replaces the current cron configuration
 
-PROJ_PATH = /home/jaleazo/projects/Spotify_Release_Pi/bash_scripts
```

Comment:
Looks like you still have this old name for the folder referenced in unit tests - I take it that's with the intention to update those later when updating unit tests?
```
git grep Spotify_Release_Pi
unit_tests/mocked_spotipy.py:sys.path.insert(1, '/home/jaleazo/projects/Spotify_Release_Pi/src')
unit_tests/test_General_Spotify_Helper.py:sys.path.insert(1, '/home/jaleazo/projects/Spotify_Release_Pi/src'
```

Reply:
Kind of, it was missed during this PR but I updated how we included the src/ directory in [BRO-90](https://github.com/jacobleazott/Spotify-2.0/pull/10).
That way we didn't have to have hard coded file paths anywhere but cron.